subprojects/docs/src/docs/userguide/publishingIvy.adoc

```diff
@@ -13,32 +13,34 @@
 // limitations under the License.
 
 [[publishing_ivy]]
-== Ivy Publishing (new)
+== The Ivy Publish Plugin
```

Comment:
Shouldn't have "The" here as plugin reference chapters should just have the name of the plugin. Not critical, though. Can be changed at a later date.